usecases/objects/references_update.go

```diff
@@ -118,20 +118,21 @@ func (req *PutReferenceInput) validate(
 	principal *models.Principal,
 	v *validation.Validator,
 	sm schemaManager, tenant string,
-) ([]*crossref.Ref, error) {
+) ([]*crossref.Ref, uint64, error) {
 	if err := validateReferenceName(req.Class, req.Property); err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 	refs, err := v.ValidateMultipleRef(ctx, req.Refs, "validate references", tenant)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 
+	// TODO-RAFT: Pull the schemaversion here and return it alongside the class
 	class, err := sm.GetClass(ctx, principal, req.Class)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
-	return refs, validateReferenceSchema(sm, class, req.Property)
+	return refs, 0, validateReferenceSchema(sm, class, req.Property)
 }
```

Comment:
**thought (if-minor):** if it's always gonna be `0` do we need to return version at all?

Reply:
i think there could be something missing to return the version 🤔

Reply:
same in usecases/objects/references_delete.go

Reply:
same in usecases/objects/references_add.go

Reply:
Discussed orally: The idea is to have the placeholder for pulling the version and then we can return it and avoid returning 0 here.

Reply:
didn't see the `// TODO-RAFT: Pull the schemaversion here and return it alongside the class`